src/state/routing/useRoutingAPITrade.ts

```diff
@@ -84,6 +84,10 @@ class GAMetric extends IMetric {
   putMetric(key: string, value: number, unit?: MetricLoggerUnit) {
     sendTiming('Routing API', `${key} | ${unit}`, value, 'client')
   }
+
+  setProperty() {
```

Comment:
This was a required update with the SOR upgrade. This is used on the BE for AWS logging but similar to `putDimensions` doesn't impact the UX.